<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>371</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_8">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>웹 사이트 주소</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="webSiteName"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>20</y>
      <width>371</width>
      <height>421</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_9">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>결과 미리보기</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="crawlResult"/>
     </item>
     <item>
      <widget class="QPushButton" name="defaultBtn">
       <property name="text">
        <string>결과창 초기화</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="crawlBtn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>502</y>
      <width>369</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>크롤링 시작</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>149</y>
      <width>371</width>
      <height>296</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>속성명(find_all)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="tag1"/>
     </item>
     <item>
      <widget class="QTextEdit" name="att1"/>
     </item>
     <item>
      <widget class="QTextEdit" name="value1"/>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>450</y>
      <width>64</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>저장 경로</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="pathBtn">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>470</y>
      <width>369</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="saveBtn">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>502</y>
      <width>369</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>저장</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>설정</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
